proxy_form_controller.js

```diff
@@ -375,7 +375,7 @@ function setConnectionProvider(providerName, serviceName) {
 // format a bytes number depending on the amount
 function formatBytes(bytes,decimals) {
   if (bytes == 0) return '0 Bytes';
-  var k = 1024,
+  var k = 1000,
 	  dm = decimals || 2,
 	  sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'],
```

Comment:
Normally kilobyte has a lowercase k, as e.g. `kB`, so I don't know if you want to bundle that change in with this PR?

Reply:
@pallas1 since you mentioned the standard, do you know if the standard is lowercase or uppercase?